specification/cdn/resource-manager/Microsoft.Cdn/preview/2022-11-01-preview/cdn.json

```diff
@@ -5220,6 +5220,18 @@
             "modelAsString": true
           }
         },
+        "isNegativeCachingEnabled": {
+          "description": "Indicates whether negative caching is enabled.",
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "ruleIsNegativeCachingEnabled",
```

Comment:
Could you change to `RuleIsNegativeCachingEnabled` with upper case `R`

Reply:
Is that right naming convention?  Seems like other name use the lowercase for first letter. https://github.com/Azure/azure-rest-api-specs/pull/22141/files#diff-9abee634d1739f3be2d6bcfbe0d17a9b304e0a0b73576a3893a04fda3393553cR5219